README.md

```diff
@@ -1,179 +1,170 @@
-## Application contract
+## Develoment
```

Comment:
What do you think about adding an H1 heading to the README with the project name and then a couple of sentence description, to soften the developer-only content here?  (I don't know if you saw the comment about the CNB readmes in https://github.com/heroku/roadmap/issues/19#issuecomment-1227680566)